llvm/test/tools/llvm-objcopy/ELF/add-invalid-note.test

```diff
@@ -0,0 +1,34 @@
+## Verify that --add-section warns on invalid note sections.
+
+# Add [namesz, descsz, type, name, desc] for a build id.
+#
```

Comment:
I suggest providing a comment before each block of echose explaining the significance of each case (i.e. what's invalid about it).

Also, make sure to use `##` for the comment.